### PR TITLE
minimum running deployment with OntoGrapher

### PR DESCRIPTION
**Pull request by MichalMed**

### CODE REVIEW

File: ontographer/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+FROM node:18 as node
```
Comment (ledsoft) [inline ontographer/Dockerfile:1]: Please, use node:16-alpine so that we use the same Node version for TermIt and OG

File: ontographer/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ENV REACT_APP_STANDALONE=$REACT_APP_STANDALONE
+
+WORKDIR /ontographer
+RUN git clone -b standalone https://github.com/datagov-cz/ontoGrapher.git .
+RUN npm install --production
+COPY . .
+RUN npx eslint --ext .ts,.tsx ./src/
+RUN npm run build
+
+FROM nginx:latest
```
Comment (ledsoft) [inline ontographer/Dockerfile:27]: Please, use nginx:1.25 so that we use the same nginx version for TermIt and OG

**Review by ledsoft — CHANGES_REQUESTED** [review]
See the inline comments.

File: README.md [hunk 3]
```diff
@@ -5,6 +5,7 @@ TermIt Docker serves to spin off a TermIt deployment, consisting of:
 - [TermIt](https://github.com/kbss-cvut/termit) (backend)
 - [Annotace](https://github.com/kbss-cvut/annotace)
 - [TermIt UI](https://github.com/kbss-cvut/termit-ui) (frontend)
+- [OntoGrapher](todo)todo
```
[the remark this answers is not in the document]
Comment (MichalMed) [reply README.md:8]: todos are to resolve after it runs on dev

File: README.md [hunk 4]
```diff
@@ -15,6 +16,7 @@ TermIt Docker serves to spin off a TermIt deployment, consisting of:
 - TermIt UI: 100MB RAM is more than enough
 - GraphDB: at least 2GB RAM (depending on the amount of data stored), 1 CPU
 - Annotace: at least 512MB RAM
+- OntoGrapher: todo
```
[the remark this answers is not in the document]
Comment (MichalMed) [reply README.md:19]: todos are to resolve after it runs on dev

File: README.md [hunk 5]
```diff
@@ -82,4 +84,5 @@ As stated above, TermIt is highly configurable. The following table lists the na
 
 The parameters are based on the [Configuration](https://github.com/kbss-cvut/termit/blob/master/src/main/java/cz/cvut/kbss/termit/util/Configuration.java) class in TermIt backend. If you need to further adjust the behavior of TermIt, consult this class.
 
-
+### OntoGrapher configuration
+ToDo
```
[the remark this answers is not in the document]
Comment (MichalMed) [reply README.md:88]: todos are to resolve after it runs on dev

**Pull request merged**